evap/evaluation/migrations/0121_add_html_body.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 3.1.7 on 2021-04-12 19:30
+
+from django.db import migrations, models
+import evap.evaluation.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('evaluation', '0120_use_django_db_models_jsonfield'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='emailtemplate',
+            name='body',
+        ),
```

Comment:
We should use RenameField and AlterField here. This RemoveField followed by AddField would delete all email templates when migrating.